var/spack/repos/builtin/packages/kokkos-kernels/package.py

```diff
@@ -6,48 +6,261 @@
 from spack import *
 
 
-class KokkosKernels(MakefilePackage):
+class KokkosKernels(Package):
     """Kokkos C++ Performance Portability Programming EcoSystem: Math Kernels -
     Provides BLAS, Sparse BLAS and Graph Kernels."""
 
     homepage = "https://github.com/kokkos/kokkos-kernels"
     url      = "https://github.com/kokkos/kokkos-kernels/archive/2.7.00.tar.gz"
+    git      = "https://github.com/kokkos/kokkos-kernels.git"
 
+    version('develop', branch='develop')
+    version('2.9.00', sha256='dafa9ebcdbcdc95641a986a80fcd3e532f1d80692c80a51cb08182c02c1added')
+    version('2.8.00', sha256='c9648dbf5b9cbddf2aa72f6d214a2c40e23e51660f903b83743617251574fe06')
     version('2.7.00', sha256='adf4af44eadbdfbeb9ec69dd5fae4e2852bd1fbe4a69213efd199e49f4098254')
-    version('2.6.00', sha256='14ebf806f66b9ca73949a478b8d959be7fa1165a640935760a724d7cc0a66335')
```

Comment:
I would still keep this old version in case someone wants to be able to reproduce the installation.